src/Identity/Core/src/SignInManager.cs

```diff
@@ -291,7 +291,7 @@ public virtual async Task<TUser> ValidateTwoFactorSecurityStampAsync(ClaimsPrinc
             {
                 return user;
             }
-            Logger.LogDebug(5, "Failed to validate a security stamp.");
```

Comment:
Looks like we have different event ids for the same message, 4 and 5 due to different methods

Reply:
@Kahbazi I think its fine to introduce the logging extensions to the API which wasn't currently using event ids, but it doesn't look like its worth the potential regression in modifying the event ids for the sign in manager/existing apis.  Unless you want to introduce a line by line method, so maybe lets just revert/delete LogginExtensions, but keep the LoggerEventIds changes

Reply:
I wanted to be consentience for each log. Should I delete `LogginExtensions` completely and add event id to each logs?
Or add another method to `LogginExtensions` for invalid security stamp?

Reply:
Right maybe the safest way to do it is https://github.com/dotnet/aspnetcore/pull/16879/files#diff-7d0bb5dd924f2d3e3de48da228cfbd19 so introduce a new file that's just the ids,, so no LoggingExtensions class like you said